pkgs/top-level/all-packages.nix

```diff
@@ -111,8 +111,6 @@ with pkgs;
   # marked with this.
   inherit (lib) dontRecurseIntoAttrs;
 
-  stringsWithDeps = lib.stringsWithDeps;
-
```

Comment:
You'll likely need to move this to become an alias with a warning first. Or at least a throw. But just removing it is a breaking change, because this was previously exposed on the package set.

Reply:
Hmm should we then remove all lib inherits from this file and use them directly like lib.removeAttrs so the aren't unexpectedly exposed?